integration/inspektor-gadget/advise_networkpolicy_test.go

```diff
@@ -35,15 +35,23 @@ func TestAdviseNetworkpolicy(t *testing.T) {
 
 	commands := []*Command{
 		CreateTestNamespaceCommand(nsServer),
-		BusyboxPodRepeatCommand(nsServer, "nc -lk -p 9090 -e /bin/cat"),
+		BusyboxPodRepeatCommand(
+			nsServer,
+			// Dual stack socket
+			"nc -lk -p 9090 -e /bin/cat",
+		),
```

Comment:
I suppose this will also work on ipv4 only systems as the traffic we're generating is not only ipv6, right?

Reply:
Yes; in this test, the traffic is ipv4.